hnn_core/network.py

```diff
@@ -299,6 +299,40 @@ def pick_connection(net, src_gids=None, target_gids=None,
     return sorted(conn_set)
 
 
+def _get_cell_index_by_synapse_type(net):
+    """Returns the indexes of excitatory and inhibitory cells in the network.
+
+    This function extracts the source GIDs (Global Identifiers) of excitatory
```

Comment:
"Global Identifiers" isn't too descriptive either ... it's a bit of a misnomer since it's mainly a programming construct. I like to say "cell ID" to be clear.